.github/workflows/slowtest.yaml

```diff
@@ -41,25 +43,23 @@ jobs:
         with:
           submodules: recursive
 
+      - name: Configure Environment
+        run: PATH="$PWD/target/debug:$PATH"
```

Comment:
I'm surprised this works. I thought one has to write it to $GITHUB_ENV to persist between step

https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#passing-values-between-steps-and-jobs-in-a-workflow

Reply:
We made this change in `test.yaml` a while back. I think it works b/c we are running all the steps on the same host... so we don't need any additional persistence. Also we are not creating a new variable, we are simply appending to `PATH` and `PATH` is always exported on the host. For it *not* to work I think github would have to do some variable resetting between steps.